We're currently migrating upstart scripts of daemons to C++. As ap-pal isn't run
as root, every daemon spawned by ap-pal isn't root either, and thus the hard
check impedes the migration. This patch removes the check if the process is run
as root.

--- a/src/igmpproxy.c	2021-10-08 08:45:12.142535564 +0800
+++ b/src/igmpproxy.c	2021-10-12 11:45:24.401153880 +0800
@@ -113,12 +113,6 @@ int main( int ArgCn, char *ArgVc[] ) {
     }
     char *configFilePath = ArgVc[optind];
 
-    // Chech that we are root
-    if (geteuid() != 0) {
-       fprintf(stderr, "igmpproxy: must be root\n");
-       exit(1);
-    }
-
     openlog("igmpproxy", LOG_PID, LOG_USER);
 
     // Write debug notice with file path...
